README.md

```diff
@@ -79,13 +79,10 @@ You can invoke this bootstap executable with the lambda-ci docker image for the
 ```sh
 # start a one-off docker container replicating the "provided" lambda runtime
 # awaiting an event to be provided via stdin
-$ unzip -o \
-    target/lambda/release/{your-binary-name}.zip \
-    -d /tmp/lambda && \
-  docker run \
+$ docker run \
     -i -e DOCKER_LAMBDA_USE_STDIN=1 \
     --rm \
-    -v /tmp/lambda:/var/task:ro,delegated \
+    -v ${PWD}/target/lambda/release:/var/task:ro,delegated \
```

Comment:
My main concern here is this might not necessarily reflect the behavior in the lambda environment. Since cargo shares on target directory for all workspace members and since more than what's packaged and delivered to the lambda environment lives in the target dir, you are like to run into cases where you'll see one behavior locally and another when running in lambda.

With the current approach you are exposing the exact same artifacts that will be deployed to lambda to your local test environment.